Neuron/Sections/Dapp/Handler/DAppTakePhotoMessageHandler.swift

```diff
@@ -43,14 +43,14 @@ class DAppTakePhotoMessageHandler: DAppNativeMessageHandler, UIImagePickerContro
                     self.webView?.viewController?.present(controller, animated: true, completion: nil)
                 }
             } else {
-                let alert = UIAlertController(title: "", message: "拍照需要相机访问权限", preferredStyle: .alert)
-                alert.addAction(UIAlertAction(title: "开启", style: .default, handler: { (_) in
+                let alert = UIAlertController(title: "", message: "DApp.DAppTakePhoto.CameraPermissions".localized(), preferredStyle: .alert)
```

Comment:
We might want to extract this as common message, as camera permission is also required from other sections. (QR scanning a wallet address when sending transaction, etc).